packages/expo/package.json

```diff
@@ -44,7 +57,7 @@
     "@clerk/shared": "2.3.3",
     "@clerk/types": "4.8.0",
     "base-64": "^1.0.0",
-    "react-native-url-polyfill": "2.0.0",
+    "react-native-url-polyfill": "^2.0.0",
```

Comment:
Usually we use specific versions, what's the reason behind this change ?

Reply:
No reason for this change, nice catch 💯